src/localization/navStrings.json

```diff
@@ -92,5 +92,27 @@
     "stocktake": "ກວດ​ກາ​ສິນ​ຄ້າ​ໃນ​ສາງ",
     "supplier_invoices": "ໃບນຳສົ່ງສິນຄ້າ",
     "supplier_requisitions": "ການ​ສະ​ເໜີ​ຂໍ ຫາຜູ້​ສະ​ໜອງ"
+  },
+  "pt": {
+    "current_stock": "Estoque atual",
+    "customer_invoices": "Faturas de clientes",
+    "customer_requisitions": "Requisições de clientes",
+    "customers": "Clientes",
+    "finalise": "FINALIZAR",
+    "finalised_cannot_be_edited": "FINALIZAR. NÃO PODE SER EDITADO",
+    "invoice": "Fatura",
+    "language": "LÍNGUA",
+    "log_out": "DESCONECTAR",
+    "manage_stocktake": "Gerenciar inventários",
+    "new_stocktake": "Novo inventário",
+    "requisition": "Requisição",
+    "stocktakes": "Inventários",
+    "stocktake": "Inventário",
+    "supplier_invoices": "Faturas de fornecedores",
+    "supplier_requisitions": "Requisições de fornecedores",
```

Comment:
can just do `  "supplier_requisitions": "Requisições de\nfornecedores",` I think - I can't see a text wrapping prop !

Reply:
Ah! Good idea. I'll try that now.

Reply:
Yep. That worked Thanks!